src/components/MenuLink.tsx

```diff
@@ -0,0 +1,23 @@
+import './MenuLink.scss';
+
+export type Props =   {
+  img : string;
+  link: string;
+  linkName: string
+}
+
+/**
+ * Creates a container with an image and a linked text
+ * @param {Props} param0 {Image to show, Link to follow, Text to show} 
+ * @returns HTML container
+ */
+const MenuLink = ({img, link, linkName} : Props) => {
+    return (
+      <div className='link_container'>
+        <img src={img}></img>&emsp;
```

Comment:
Since you are working with SVGs, I'd recommend creating react components around them and importing them as react components. For example:

```tsx
const ClockIcon = () => {
    return (<svg>
    ...
    </svg>)
}
```

And importing those components directly

Reply:
On commit [4d8acad](https://github.com/TheTributeCommunity/fesbal-frontend/pull/40/commits/4d8acada68f7efa34be3510737da2a0263f55623) SVGs were created as React components, but im not sure if this is correct. Right now the whole svg information is written in each file, and i think that if we needed to change that SVG, we would need to modify that info and no just change the file with the drawing.
Should this be changed to be a React component that returns an image element with the source of the svg? or is it okay as it is?